src/utils.ts

```diff
@@ -138,10 +138,11 @@ export function getMessageFromGraphQLError(error: Error): string {
  */
 export const responseBuilder = (
   code = 200,
-  message?: string,
+  message?: string | Record<string, unknown>,
```

Comment:
@jesusrodrz it seems that the best approach here is for this to be an object. 

If that's the case let's either remove the string support or create a new version of the function.

Reply:
?